internal/pkg/manifest/env.go

```diff
@@ -119,11 +119,11 @@ type environmentNetworkConfig struct {
 }
 
 type environmentVPCConfig struct {
-	ID                  *string              `yaml:"id,omitempty"`
-	CIDR                *IPNet               `yaml:"cidr,omitempty"`
-	Subnets             subnetsConfiguration `yaml:"subnets,omitempty"`
-	SecurityGroupConfig securityGroupConfig  `yaml:"security_group,omitempty"`
-	Flowlogs            *bool                `yaml:"flow_logs,omitempty"`
+	ID                  *string                      `yaml:"id,omitempty"`
+	CIDR                *IPNet                       `yaml:"cidr,omitempty"`
+	Subnets             subnetsConfiguration         `yaml:"subnets,omitempty"`
+	SecurityGroupConfig securityGroupConfig          `yaml:"security_group,omitempty"`
+	FlowLogs            Union[bool, VPCFlowLogsArgs] `yaml:"flow_logs,omitempty"`
```

Comment:
I saw this change in @dannyrandall's PR:
https://github.com/aws/copilot-cli/pull/4150/files#diff-d5ce3a84f02de238dd26290c8a7fcf956134998ff644184f5d13a7e692eda1bbR30

I think we'd need one as well in `transform.go` but I'm not fully certain. I would love finding a way such taht using the `Union` type doesn't result in having to modify `transform.go`